src/translations/en.json

```diff
@@ -223,6 +223,11 @@
       "service": {
         "run": "Run"
       },
+      "update": {
+        "updating": "Updating",
```

Comment:
Should this be "Installing" instead?

Reply:
Yeah maybe... I called it installing, based on the change/discussions we made some time ago with ESPHome. Downloading, uploading firmware, installing, updating, flashing.

As we are dealing with software, firmware and whatever else, I figured it would be the same context. Thus the component uses "install".

Reply:
Changed to Installing in https://github.com/home-assistant/frontend/commit/79bf80360541c86f6eacecbed5edd3a830d9f739